src/backend/webGlObjects/baseWebGlObject.ts

```diff
@@ -7,6 +7,9 @@ export type WebGlObjectTag = {
 
 export class WebGlObjects {
     public static getWebGlObjectTag(object: WebGLObject): WebGlObjectTag {
+        if (!object) {
```

Comment:
how do we end up in this situation ?

Reply:
The version as written is erroneous and is too wide a check; it should be `if (object === undefined || object === null) {` -- which happens sometimes upon a `readFromContext` call from within headset only. I'll need to verify tomorrow once I have access to my headset again.

(As far as I know, `readFromContext` is the only place we can end up with a null object, since all other uses of `getWebGlObjectTag` have null checks against the object before running the function.)

Reply:
Confirmed that it is from a `readFromContext` call, and the parameters that are null are `IMPLEMENTATION_COLOR_READ_FORMAT` and `IMPLEMENTATION_COLOR_READ_TYPE`. Note that this case only occurs when using WebXR layers, and only from within headset.